packages/core-flows/src/definition/cart/workflows/update-carts.ts

```diff
@@ -0,0 +1,20 @@
+import {
+  CartDTO,
+  FilterableCartProps,
+  UpdateCartDataDTO,
+} from "@medusajs/types"
+import { WorkflowData, createWorkflow } from "@medusajs/workflows-sdk"
+import { updateCartsStep } from "../steps/update-carts"
+
+type WorkflowInput = {
```

Comment:
suggestion: interface, plus same question/comment as for phil pr's in terms of location